Source/options.cpp

```diff
@@ -55,7 +55,7 @@ constexpr OptionEntryFlags OnlyIfNoImplicitRenderer = OptionEntryFlags::Invisibl
 constexpr OptionEntryFlags OnlyIfNoImplicitRenderer = OptionEntryFlags::None;
 #endif
 
-#if defined(__ANDROID__) || defined(TARGET_OS_IPHONE)
+#if defined(__ANDROID__) || TARGET_OS_IPHONE

```

Comment:
> Identifiers that are not macros, which are all considered to be the number zero. This allows you to write #if MACRO instead of #ifdef MACRO, if you know that MACRO, when defined, will always have a nonzero value. Function-like macros used without their function call parentheses are also treated as zero. 

This is a nice shortcut but will cause warnings with certain compiler flags.

> In some contexts this shortcut is undesirable. The '-Wundef' option causes GCC to warn whenever it encounters an identifier which is not a macro in an #if.

Perhaps it'd be better to test for `(defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE == 1)`?

Reply:
Good point, I'm not sure if we have -Wundef on (the MSVC equivalent isn't active) but it's trivial to avoid.